upgrade/1.2/plan_and_coordinate_network_upgrade.md

```diff
@@ -0,0 +1,152 @@
+
+   During the CSM 1.2 upgrade, UANs will retain their pre-1.2 CAN IP addresses in order to minimize disruption to UANs. However,
+   toward the end of the CSM 1.2 upgrade, UANs will stop registering themselves on CMN and will receive new IP addresses on the
+   CAN/CHN network. This process is described in more detail in [UAN Migration](#uan-migration).
+
+   Pivoting the pre-1.2 CAN to the new CMN allows administrative traffic (already on the pre-1.2 CAN) to remain as-is while
+   moving standard user traffic to a new site-routable network (CAN / CHN).
+
+1. Customer Access Network (CAN) / Customer High-speed Network (CHN)
+
+   For user traffic only (e.g. users running and monitoring jobs), CSM 1.2 allows a choice of one of two networks:
+
+    - Customer Access Network (CAN) \[Recommended\]: this is a new network (VLAN6 in switches) that runs over the management network. This
+      network must not be confused with pre-1.2 CAN, which was a monolithic network that allowed both user and administrative
+      traffic, was configured as VLAN7 in switches, and is now renamed to CMN. The new CAN allows only user traffic.
+
+    - Customer High-speed Network (CHN) \[CSM 1.2 Tech Preview\]: this is a new network (VLAN5 in switches) that runs over the high-speed fabric.
+
+   Either the new CAN or CHN must be chosen, but not both. Note that the CHN is a technical preview in CSM 1.2, and the new CAN is
+   the recommended upgrade. The rest of the upgrade guide provides options for configuring either the new CAN or CHN.
+
+## UAN migration
+
+Steps are taken in order to minimize disruption to UANs during the CSM 1.2 upgrade process. Read these steps
+carefully and follow any recommendations and warnings to minimize disruptions to user activity. Note that these steps apply
+to all types of application nodes and not just UANs -- the term "UAN" just happens to be more commonly used and understood when
+referring to user activity.
+
+1. During the upgrade, the switch `1.2 Preconfig` will not remove UAN ports from the CMN VLAN (the pre-1.2 CAN), allowing UANs
+   to retain their existing IP addresses during the CSM 1.2 upgrade process. Traffic to and from UANs will still flow through CMN, but
+   may also flow through CAN/CHN networks, if desired.
+
+1. CFS will be temporarily disabled for UANs, in order to prevent CFS plays from removing CMN interfaces from UANs. Note that network
+   configuration is controlled by data in SLS but CFS plays also pick up the same SLS data, which can lead to UANs being prematurely
+   removed from the CMN and causing UAN outages. As such, CFS plays need to be disabled for UANs.
+
+   To disable CFS plays for UANs, remove CFS assignment for UANs by running the following command:
+
+   ```bash
+   ncn-m001# for xname in $(cray hsm state components list --role Application --subrole UAN --type node --format json | jq -r .Components[].ID) ; do
```

Comment:
Remove the prompts to follow the main conventions https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts-inside-of-directions

This comment applies to every prompt.